plugins/csv-import/src/csv.ts

```diff
@@ -97,66 +108,98 @@ const findRecordValue = (record: CSVRecord, key: string) => {
 const collator = new Intl.Collator("en", { sensitivity: "base" })
 const BOOLEAN_TRUTHY_VALUES = /1|y(?:es)?|true/iu
 
-function getRecordValueForField(
-    field: CollectionField,
+function getFieldInputForField(
+    field: Field,
     value: string | null,
     allItemIdBySlug: Map<string, Map<string, string>>
-) {
-    if (value === null) {
-        return undefined
-    }
-
+): FieldDataEntryInput | ConversionError {
     switch (field.type) {
         case "string":
+            if (value === null) {
+                return { type: "string", value: "" }
+            }
+            return { type: "string", value }
+
         case "formattedText":
-        case "link":
+            if (value === null) {
+                return { type: "formattedText", value: "" }
+            }
+            return { type: "formattedText", value }
+
         case "color":
+            return { type: "color", value }
+
+        case "link":
+            return { type: "link", value }
+
         case "file":
+            return { type: "file", value }
+
         case "image":
-            return value.trim()
+            return { type: "image", value }
 
         case "number": {
+            if (value === null) {
+                return { type: "number", value: 0 }
+            }
             const number = Number(value)
             if (Number.isNaN(number)) {
                 return new ConversionError(`Invalid value for field “${field.name}” expected a number`)
             }
-            return number
+            return { type: "number", value: number }
         }
 
-        case "boolean":
-            return BOOLEAN_TRUTHY_VALUES.test(value)
+        case "boolean": {
+            if (value === null) {
+                return { type: "boolean", value: false }
+            }
+            return { type: "boolean", value: BOOLEAN_TRUTHY_VALUES.test(value) }
```

Comment:
`BOOLEAN_TRUTHY_VALUES.test(null)` is `false`

```suggestion
            return { type: "boolean", value: BOOLEAN_TRUTHY_VALUES.test(value) }
```

Reply:
RegExp#test only allows a string :( TypeScript not

Reply:
```suggestion
            return { type: "boolean", value: value ? BOOLEAN_TRUTHY_VALUES.test(value) : false }
```

Reply:
Ah ok - yeh looks good